cmd/istioctl/main.go

```diff
@@ -271,7 +300,7 @@ func init() {
 	rootCmd.AddCommand(putCmd)
 	rootCmd.AddCommand(getCmd)
 	rootCmd.AddCommand(deleteCmd)
-	rootCmd.AddCommand(injectCmd)
+	// rootCmd.AddCommand(injectCmd)
```

Comment:
inject.go and any other platform specific commands can themselves  to rootCmd in their own init() function. We can use build tags to selectively include the platform specific commands in the build if that is needed.